Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PauseMenu : MonoBehaviour {
+    public static bool GameIsPaused = false;
+
+    public GameObject pauseMenuUI;
+    // Update is called once per frame
+    void Update()
+    {
```

Comment:
This should be converted to use the new unity input system.

Reply:
Here is a video showing how to read input from the new system https://www.youtube.com/watch?v=3zEpfMbE30s
You can also find an example in PlayerController.cs